requirements.txt

```diff
@@ -1,7 +1 @@
-backports.shutil_get_terminal_size==1.0.0;python_version < '3.3'
-log_symbols==0.0.11
-spinners==0.0.23
-cursor==1.2.0
-termcolor==1.1.0
-colorama==0.3.9
-six==1.11.0
+--editable .
```

Comment:
I don't think we need this file now

Reply:
Aye, I'd concur with this and your [other](https://github.com/manrajgrover/halo/pull/105/files/e7f111954313147a4eaaa55071e0b52f12bbca27#r228761593) [comments](https://github.com/manrajgrover/halo/pull/105/files/e7f111954313147a4eaaa55071e0b52f12bbca27#r228761572). I think I kept 'em in for historical reasons, or to make as few procedural changes as possible. I'm not really sure why. But I do agree — I'll remove them.